src/cf-forms/usage.md

```diff
@@ -22,39 +22,44 @@ Capital Framework.
     - [Label helper text](#label-helper-text)
 - [Inputs](#inputs)
     - [Basic Text Inputs](#basic-text-inputs)
-        - [Default state](#default-state)
-        - [Hover state](#hover-state)
-        - [Focus state](#focus-state)
-        - [Disabled state](#disabled-state)
+        - [Text input default state](#text-input-default-state)
+        - [Text input hover state](#text-input-hover-state)
+        - [Text input focus state](#text-input-focus-state)
+        - [Text input disabled state](#text-input-disabled-state)
         - [Full-width modifier](#full-width-modifier)
     - [Basic checkboxes](#basic-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Checkbox default state](#checkbox-default-state)
```

Comment:
I recommend that we don't repeat "checkbox" for all of these. The "Basic checkboxes" can cover this for the whole set. If we want to match the DM we could include a "States" heading and then not repeat the word state down the list (instead: States - Default, Hover, Focus, Selected, Disabled).

Reply:
The problem is these are links to unique headings on the page, if they are all e.g. "Default state" all the links for each input in the table of contents will link to the first occurrence, so they will all link to text inputs.

Reply:
Not sure if it's possible in markdown to have the link and the text for the link different. If you know a place in DM that does this let me know!